src/components/Users/Users.js

```diff
@@ -47,8 +49,44 @@ const Users = (props) => {
       </div>
       <div>
         {user.followed
-          ? <button onClick={() => props.unfollow(user.id)}>Unfollow</button>
-          : <button onClick={() => props.follow(user.id)}>Follow</button>}
+
+          ? (
+            <button onClick={() => {
+              axios.delete(`${USERS_URL}follow/${user.id}`, {
+                withCredentials: true,
+                headers: {
+                  'API-KEY': 'a330ffbb-7ace-4dc9-a8ec-7880ecab78ff',
+                },
+              })
+                .then((response) => {
+                  if (response.data.resultCode === 0) {
+                    props.unfollow(user.id);
+                  }
+                });
+            }}
+            >
+Unfollow
+            </button>
+          )
+
+          : (
+            <button onClick={() => {
```

Comment:
Consider moving the onClick handler out of JSX.

Reply:
/fxd